Sources/UINotificationView.swift

```diff
@@ -168,8 +168,8 @@ open class UINotificationView: UIView {
         imageViewHeightConstraint = imageView.heightAnchor.constraint(equalToConstant: 31)
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/UINotificationView.swift;
  Line: 169;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)